src/pages/snap/{Snap.location}/{Snap.slug}.test.tsx

```diff
@@ -24,6 +33,8 @@ describe('Snap page', () => {
   it('renders', async () => {
     mockUseAccount.mockReturnValue({ isConnected: true });
 
+    mockUseAccount.mockReturnValue({ isConnected: true });
```

Comment:
it looks like you doubled the statement for all of below?

Reply:
:( that was done after rebase, i will removed all of this.